skymp5-server/cpp/addon/ScampServer.cpp

```diff
@@ -150,6 +150,16 @@ ScampServer::ScampServer(const Napi::CallbackInfo& info)
 
     auto serverSettings = nlohmann::json::parse(buffer.str());
 
+    if (serverSettings.find("weaponStaminaModifiers") !=
```

Comment:
I think that we should add more logging here.

so we let our team know what values server uses.